Extensions/DataSources/.gitignore

```diff
@@ -0,0 +1,4 @@
+npm-debug.log
```

Comment:
We shouldn't add .gitignore .. if files are being generated here, then we might need to tweak build process a bit.

[Comment to be taken later]